packages/gatsby-transformer-toml/src/gatsby-node.js

```diff
@@ -0,0 +1,45 @@
+const toml = require(`toml`)
+const _ = require(`lodash`)
+const crypto = require(`crypto`)
+
+async function onCreateNode({ node, boundActionCreators, loadNodeContent }) {
+  const { createNode, createParentChildLink } = boundActionCreators
+  // Filter out non-toml content
+  // For some reason TOML files are considered 'application/octet-stream' in 'mime'
```

Comment:
it's just that toml isn't a known extension yet (too new / unused perhaps) https://github.com/jshttp/mime-db